README.md

```diff
@@ -69,7 +69,7 @@ Or even rebuild the original document from its predictions:
 ```python
 import matplotlib.pyplot as plt
 
-plt.imshow(result.synthesize()); plt.axis('off'); plt.show()
+plt.imshow(result.synthesize()[0]); plt.axis('off'); plt.show()
```

Comment:
Was already fixed in #590 :sweat_smile: Mind reversing that change please? :pray: